src/dgipy/dgidb.py

```diff
@@ -373,7 +484,7 @@ def __process_gene_search(results: dict) -> pd.DataFrame:
     pmids_list = []
     # genecategories_list = []
 
-    for match in results["data"]["genes"]["nodes"]:
+    for match in results["genes"]["nodes"]:
```

Comment:
outside of scope of this PR, but fwiw I am 100% open to refactoring this result table building method. Pretty sure there is a better way to do this, this is just what I got down at the time